src/applications/pre-need/components/IntroductionPage.jsx

```diff
@@ -27,6 +27,7 @@ class IntroductionPage extends React.Component {
           Eligibility for Burial in a VA National Cemetery).
         </p>
         <SaveInProgressIntro
+          downtime={this.props.route.formConfig.downtime}
```

Comment:
in order to see the downtime notification, you have to pipe the downtime config into the `SaveInProgressIntro` component. It uses the DowntimeNotification component which requires a start and end time to work (pulled from the state).